dataTW.py

```
@@ -0,0 +1,67 @@
+import tweepy as tw
+from tweepy.auth import OAuthHandler
+import csv
+import urllib2
+import re
+import os
+from dotenv import load_dotenv, find_dotenv
+from config import tweetsId, tweets_user
+
+
+load_dotenv(find_dotenv())
+
+
+CONSUMER_KEY = os.getenv('CONSUMER_KEY')
+CONSUMER_SECRET = os.getenv('CONSUMER_SECRET')
+ACCESS_TOKEN = os.getenv('ACCESS_TOKEN')
+ACCESS_SECRET = os.getenv('ACCESS_SECRET')
+
+
+auth = tw.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+auth.set_access_token(ACCESS_TOKEN, ACCESS_SECRET)
+
+api = tw.API(auth)
+
+
+def get_user_ids_of_post_likes(post_id):
+    try:
+        json_data = urllib2.urlopen(
+            'https://twitter.com/i/activity/favorited_popup?id=' + str(post_id)).read()
+        found_ids = re.findall(r'data-user-id=\\"+\d+', json_data)
+        unique_ids = list(set([re.findall(r'\d+', match)[0]
+                               for match in found_ids]))
+        return unique_ids
+    except urllib2.HTTPError:
```

it's probably better to let this crash if there's an error, so you can see what is going on exactly, instead of it "failing silently"